thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -301,17 +319,28 @@ export default Ember.Controller.extend({
     const startTime = moment().subtract(1, 'month').endOf('day').utc().format("YYYY-MM-DDTHH:mm:ss.SSS[Z]");
     const endTime = moment().subtract(1, 'day').endOf('day').utc().format("YYYY-MM-DDTHH:mm:ss.SSS[Z]");
 
+    // Set banner to 'pending' state
+    this.setProperties({
+      isReplaySuccess: true,
+      isReplayStatusPending: true
+    });
+
+    // Begin replay sequence. Simulate trigger response time since we don't yet need
+    // to wait for the actual response, which takes 30+ seconds
     this.callCloneAlert(newFuncId)
       .then((clonedId) => {
+        const that = this;
+        Ember.run.later((function() {
```

Comment:
Yes, that was intentional, but I'm modifying it after today's meeting (replay sequence is now being modified)